src/v2/Apps/Home/Components/HomeRecentlyViewedRail.tsx

```diff
@@ -0,0 +1,107 @@
+import { Box, Shelf, Skeleton, SkeletonText, SkeletonBox } from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { useSystemContext } from "v2/System"
+import { SystemQueryRenderer } from "v2/System/Relay/SystemQueryRenderer"
+import { HomeRecentlyViewedRail_homePage } from "v2/__generated__/HomeRecentlyViewedRail_homePage.graphql"
+import { HomeRecentlyViewedRailQuery } from "v2/__generated__/HomeRecentlyViewedRailQuery.graphql"
+import { ShelfArtworkFragmentContainer } from "v2/Components/Artwork/ShelfArtwork"
+
+interface HomeRecentlyViewedRailProps {
+  homePage: HomeRecentlyViewedRail_homePage
+}
+
+const HomeRecentlyViewedRail: React.FC<HomeRecentlyViewedRailProps> = ({
+  homePage,
+}) => {
+  const results = homePage.artworkModule?.results
+  if (!results || results?.length === 0) {
+    return null
+  }
+
+  return (
+    <Shelf>
+      {results.map((artwork, index) => {
+        return (
+          <ShelfArtworkFragmentContainer
+            artwork={artwork!}
+            key={index}
+            // TODO: Add home type to cohesion once we have tracking
+            contextModule={null as any}
+            hidePartnerName
+            lazyLoad
+          />
+        )
+      })}
+    </Shelf>
+  )
+}
+
+const PLACEHOLDER = (
+  <Skeleton>
+    <Shelf>
+      {[...new Array(8)].map((_, i) => {
+        return (
+          <Box width={325} key={i}>
+            <SkeletonBox width={325} height={230} />
+            <SkeletonText variant="lg">Some Artist</SkeletonText>
+            <SkeletonText variant="md">Location</SkeletonText>
+          </Box>
+        )
+      })}
+    </Shelf>
+  </Skeleton>
+)
```

Comment:
It looks like this placeholder was copied over from a different style rail, doesn't match artwork shelves

Reply:
Fixed